draco/analysis/transform.py

```diff
@@ -433,6 +433,12 @@ class MModeTransform(task.SingleTask):
 
     Currently ignores any noise weighting.
     """
+    def setup(self, manager=None):
```

Comment:
The docstring for this function should explain the behaviour here.

Reply:
You might also want to explain the behaviour in the class docstring. Only one location needs a full account though.